my_list.rb

```diff
@@ -0,0 +1,11 @@
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
```

Comment:
- [ ] Please make sure to require your `my_list` class here. Currently, is throwing this error:

![image](https://github.com/alexansaa/Ruby-Enumerable/assets/77069011/ef7ba5e6-e2f6-43a5-97b4-ee90283aee53)

```suggestion
require_relative 'my_enumerable'

class MyList
  include MyEnumerable

  def initialize(*list)
    @list = list
  end

  def each(&block)
    @list.each(&block)
  end
end
```


Please make sure to fix it 🤓